### ISSUE TITLE
Snort control socket

### ISSUE THREAD

**Issue by malvads** [snort3/snort3#385]
Hi, I now that in snort2 you can build it with control socket option which allows to send data to the snort daemon via unix socket, rn im building some features for my snort3 and I need to send some commands to the snort daemon to reload some custom things in the fly that I'm building rn, the question is, this is supported in snort3?, if so, there is any documentation that I can check?

thx

**Comment by matirko** [comment]
Snort3 allows you to interface with it via control socket. This is configured via a command line option:
```
--control-socket <file> to create unix socket
```

(See [the snort reference](https://github.com/snort3/snort3/releases/download/3.3.7.0/snort_reference.html) for more details on this)

You will have to already have shell support in your Snort binary. If this isn't the case, during the `configure` step, make sure to pass in `--enable-shell` - this is discussed more in section 11.1 (Build Options) in the reference guide.

In short, the steps are: build Snort with `--enable-shell`, make sure to specify your socket when you startup Snort with `--control-socket <file>`, and then connect to your socket with `nc -U <snort socket file>`

Commands are listed per inspector in the reference guide as well.

**Comment by malvads** [comment]
thx, I was able to connect to daemon :)
